lib/std/macho.zig

```diff
@@ -1900,6 +1908,9 @@ pub const LoadCommandIterator = struct {
         hdr: load_command,
         data: []const u8,
 
+        const section_ = if (builtin.abi == .ilp32) section else section_64;
+        const segment_command_ = if (builtin.abi == .ilp32) segment_command else segment_command_64;
```

Comment:
This parsing code should not be dependent on the target platform. It's intended to be usable on any target, just like `std.coff`, `std.elf`, etc.